src/gamestate/dcon_generated.txt

```diff
@@ -4516,6 +4516,11 @@ object {
 		name{ auto_choice }
 		type{ uint8_t }
 	}
+	property{
```

Comment:
We don't need to store this for every event. Instead you can just store it for the set of election events (i.e. store a struct in the vector of election events instead with the issue id in there as well)